doc/script_commands.txt

```diff
@@ -155,6 +155,7 @@ is given as "--ja--", the 'japanese name' field from the monster database is
 used, (which, in rAthena, actually contains an English name) if it's "--en--", 
 it's the 'english name' from the monster database (which contains an uppercase 
 name used to summon the monster with a GM command).
+Monster level will change the level of the monsters spawn (optionnal).
```

Comment:
`<monster level> can be used to change the spawned monster's level from the database value (optional).`